### PR TITLE
Add clarification about MSI write replaced by MRIF update

### PR DESCRIPTION
**Pull request by avpatel**

### CODE REVIEW

File: doc/src/IOMMU.tex [hunk 1]
```diff
@@ -871,24 +871,25 @@ \subsection{MSI PTE, MRIF mode}
...
+A non-atomic read-modify-write sequence, if non-atomic updates are supported
 
 \end{enumerate}
-The write of step~1 must be visible to all agents in the system before
-the write of step~2 becomes visible to any agent.
+Once the MRIF update operation is visible to all agents in the system,
+the \mbox{11-bit} NID value is zero-extended to 32~bits, and this value
+is written to the address NPPN\,\z{<<}\,12 (i.e., physical page number NPPN,
+page offset zero)
+in little-endian byte order. The write of step~1 must be visible to all agents
```
Comment (dkruckemyer-ventana) [inline doc/src/IOMMU.tex:891]: The last sentence in this paragraph has stale references.

Comment (avpatel) [reply doc/src/IOMMU.tex]: Removed.

**Comment by ved-rivos** [comment]
Would also be good to add the following clarifications:
The memory write to set the pending bit in MRIF must follow the ordering rules as applicable to the original MSI write. For PCIe, for example, this implies that a) the memory write to set the pending bit must not be visible to any agent before all previous posted requests are visible to all agents and b) later read completion sent by that device must not be delivered before the write to set the pending bit is visible to all agents.

**Comment by jhauser-us** [comment]
@ved-rivos, I disagree.  Topics beyond the scope of the AIA include details of ordering rules for transactions over PCIe or any other specific buses, and most details about synchronizing software to IOMMU-relevant events, which I understand is part of you folks' original motivation for these clarifications.  If you believe you need these additional requirements, you can put them in the RISC-V IOMMU spec.  They are too IOMMU-specific and bus-specific for the AIA.

**Pull request merged**